gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Option.java

```diff
@@ -69,4 +73,20 @@ public String toString() {
         .add("value", value)
         .toString();
   }
+
+  static String selector(List<FieldSelector> required, FieldSelector... others) {
```

Comment:
This also could be done in a common place (e.g. provide a helper class inside the `FieldSelector` interface).  did you consider it?